javascript-arrays/main.js

```diff
@@ -0,0 +1,26 @@
+var colors = ['red', 'white', 'blue'];
+console.log('value of colors[0]:', colors[0]);
+console.log('value of colors[1]:', colors[1]);
+console.log('value of colors[2]:', colors[2]);
+
+var america = 'America is ' + colors[0] + ', ' + colors[1] +
+              ',and ' + colors[2] + '.';
+console.log(america);
+
+colors[2] = 'green';
+
+var mexico = 'Mexico is ' + colors[0] + ', ' + colors[1] +
+            ',and ' + colors[2] + '.';
+console.log(mexico);
+console.log('value of colors:', colors);
+
+var students = ['Leonardo', 'Donatello', 'Raphael', 'Michelangelo'];
+var numberOfStudents = students.length;
+var conLog1 = 'There are ' + numberOfStudents + ' students in the class.';
+console.log(conLog1);
+
+var lastIndex = numberOfStudents - 1;
+var laststudent = students[lastIndex];
```

Comment:
small thing: just make sure to use `camelCase` for your variable names, makes it easier to read